lib/notification/adapter/http.md

```diff
@@ -0,0 +1,31 @@
+### HTTP Adapter
+
+This is a generic adapter for sending notifications via HTTP requests. 
```

Comment:
You're not mentioning the auth token here. I'd make it optional, but describe how to use it in any case. Note, not all users are pro's ;)